atlasdb-ete-tests/src/main/java/com/palantir/atlasdb/AtlasDbEteServer.java

```diff
@@ -77,18 +79,20 @@ public void run(AtlasDbEteConfiguration config, final Environment environment) t
     private TransactionManager tryToCreateTransactionManager(AtlasDbEteConfiguration config, Environment environment)
             throws InterruptedException {
         if (config.getAtlasDbConfig().initializeAsync()) {
-            return createTransactionManager(config.getAtlasDbConfig(), environment);
+            return createTransactionManager(config.getAtlasDbConfig(), environment, config.getAtlasDbRuntimeConfig());
         } else {
-            return createTransactionManagerWithRetry(config.getAtlasDbConfig(), environment);
+            return createTransactionManagerWithRetry(config.getAtlasDbConfig(), environment,
+                    config.getAtlasDbRuntimeConfig());
         }
     }
 
-    private TransactionManager createTransactionManagerWithRetry(AtlasDbConfig config, Environment environment)
+    private TransactionManager createTransactionManagerWithRetry(AtlasDbConfig config, Environment environment,
```

Comment:
For these methods it's super weird to have the environment parameter sandwiched in between two config parameters.